config/config.exs

```diff
@@ -33,6 +33,14 @@ config :phoenix, :generators,
   migration: true,
   binary_id: false
 
+config :mime, :types, %{
+  "*/*" => ["json"],
+  "application/json" => ["json"],
+  "application/binary" => ["binary"],
+  "application/avro.binary" => ["avro"],
```

Comment:
binary: when client specifies "Accept: application/binary" header, KastleX will return value of the message at the given offset in response body and will not try to format it in json.

avro.binary: for the future, the idea is to decode messages in KastleX and return them as json (the better name probably would be "decoded-avro-binary" or something like this).